coopr-docs/docs/source/rest/metrics.rst

```diff
@@ -81,3 +81,149 @@ Example
            "total": 1
        }
    }
+
+
+
+Get Node Usage Metrics
+======================
+
+To get node usage metrics, HTTP GET request to URI:
+::
+
+/metrics/nodes/usage
+
+Only admins are allowed to get node usage metrics. Tenant admins will see metrics for their own nodes,
+whereas the superadmin will get metrics for all tenant nodes.
```

Comment:
Only admins are allowed to retrieve node usage metrics. Tenant admins will see metrics for their own nodes,
while the superadmin will see metrics for all tenant nodes.